src/Stryker.Core/Stryker.Core/Mutants/CsharpMutantOrchestrator.cs

```diff
@@ -51,7 +51,8 @@ public CsharpMutantOrchestrator(IEnumerable<IMutator> mutators = null, StrykerOp
                 new NullCoalescingExpressionMutator(),
                 new MathMutator(),
                 new SwitchExpressionMutator(),
-                new IsPatternExpressionMutator()
+                new IsPatternExpressionMutator(),
+                new StringIsNullOrMutator()
```

Comment:
StringIsNullOrEmptyMutator? Also, I believe this should be just part of the StringEmptyMutator somehow.

Reply:
https://github.com/stryker-mutator/stryker-net/blob/master/src/Stryker.Core/Stryker.Core/Mutators/MutatorBase.cs#L11

:) If it makes more sense, you can implement IMutator directly.

Reply:
Refactored to `StringEmptyMutator`